projects/packages/search/src/customberg/index.jsx

```diff
@@ -20,4 +20,4 @@ function initialize( id ) {
 	root.render( <Layout /> );
 }
 
-global.jetpackSearchConfigureInit = initialize;
+window.jetpackSearchConfigureInit = initialize;
```

Comment:
"global" is node, not browser. Webpack turns the "global" into "window".

Reply:
Funny enough, I looked that up just before I hit this comment.